Library/Formula/camlp5.rb

```diff
@@ -13,20 +16,36 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
```

Comment:
Is it confusing to have `--with-doc` and `--with-tex` when they both are options to install docs? Should it be `--with-html` and `--with-tex` instead?